.github/workflows/check-image-vulnerabilities.yml

```diff
@@ -32,17 +26,51 @@ jobs:
           } >> "$GITHUB_STEP_SUMMARY"
 
   check-image-vulnerabilities:
-    name: Checks images in version ${{ inputs.version }} for vulnerabilities
+    name: Check images in version ${{ inputs.version }} for vulnerabilities
     runs-on: ubuntu-latest
     env:
-      QUAY_BEARER_TOKEN: ${{ secrets.QUAY_RHACS_ENG_VULNERABILITY_BEARER_TOKEN }}
+      # ACS-CS dogfooding instance.
+      ROX_ENDPOINT: "acs-cii18grublkv81uil8gg.acs.rhcloud.com"
+    permissions:
+      id-token: write
+    strategy:
+      matrix:
+        image:
+          [
+            "central-db",
+            "collector",
+            "collector-slim",
+            "main",
+            "roxctl",
+            "scanner",
+            "scanner-db",
+            "scanner-db-slim",
+            "scanner-slim",
+            "stackrox-operator",
+          ]
```

Comment:
```suggestion
        - "central-db"
        - "collector"
        - "collector-slim"
        - "main"
        - "roxctl"
        - "scanner"
        - "scanner-db"
        - "scanner-db-slim"
        - "scanner-slim"
        - "stackrox-operator"
```

Reply:
What's the advantage of this change? I prefer the list formatting.

Reply:
more like yaml, less like JSON - I find it easier to read. 

Where are `scanner-v4` images though?